.gitignore

```diff
@@ -12,3 +12,6 @@ debug
 *.pprof
 .DS_Store
 \.project
+dist/
+node_modules/
+package-lock.json

```

Comment:
I was under the impression package-lock.json was supposed to be checked in.  https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
It probably should, but there are no production dependencies, so I wasn't sure how important it was.

Reply:
True.  It's a mammoth file and it's likely to change a lot before initial release, so waiting is probably a good idea.